app/models/project.server.ts

```diff
@@ -197,121 +210,143 @@ export async function getProjects(where: ProjectWhereInput) {
         skills: true,
         projectMembers: true,
       },
-    })
-    return projects
+    });
+    return projects;
   } catch (e) {
-    throw new Error(JSON.stringify(e))
+    throw new Error(JSON.stringify(e));
   }
 }
 
 export const validateIsTeamMember = (
```

Comment:
What do you think if we change this validateIsTeamMember and  for "validateHasPermissions" and we add the isAdmin business rule currently we have:

```
 if (!isAdmin)
    validateIsTeamMember(profileId, projectMembers, project.ownerId);
```

And inside validateIsTeamMember we are really checking permissions, because the function does not validate if it is team member it validates if has permissions.

Comment and function don't correspond

Proposal:

```
 if (!isAdmin) // remove this if
    validatePermission(isAdmin, profileId, projectMembers, project.ownerId);

and in  

validateUserPermission(isAdmin, profileId, projectMembers, project.ownerId){ 
//if is admin return true
//perform bussinnes rules for membership and ownership
}
```